src/thunks/fetchCountries.test.js

```diff
@@ -147,14 +149,27 @@ describe('fetchCountries', () => {
   // });
 
   it('should dispatch setCountries with the correct params', async () => {
-    jest.mock('./fetchCountries')
+    window.fetch = jest.fn().mockImplementation(() => Promise.resolve({
+      ok: false,
+      statusText: 'Something went wrong'
+    }))
+    const setCountries = jest.fn().mockImplementation((data) => true )
+
+    let mockDispatch = jest.fn().mockImplementation((fnc) =>  fnc )
+    const fetchCountries = jest.fn().mockImplementation ((url) =>  async (mockDispatch) => {
+        const response = await fetch(url)
+      
+        const data = await response
+        // dispatch(isLoading(false))
+        console.log(setCountries())
+        mockDispatch(setCountries(data))
+      })
+
```

Comment:
I can't tell you how happy I am that you figured this out. By mocking everything out, I guess we made sure to have a thorough test. Great work!